components/alert/README.md

```diff
@@ -50,7 +50,6 @@ a pop-up at the bottom of the screen that automatically dismisses itself by defa
 - `no-auto-close` (Boolean, default: `false`) prevents the alert from automatically closing 2.5 seconds after opening.
 - `open` (Boolean, default: `false`): open or close the toast alert.
 - `subtext` (optional, String) The text that is displayed below the main alert message.
-- `type` (String, default: `'default'`): type of the alert being displayed. Can be one of  `default`, `critical`, `success` , `warning`
```

Comment:
So despite the design saying this, there are definitely several places that are using toasts of different colours. I specifically noticed [one in quizzing](https://search.d2l.dev/xref/lms/le/quizzing/webpages/user/attempt/quiz_attempt_page_auto.js?r=38b26bb2#227) during my unbundling work where it's using a red and green toast for "Connect Lost" / "Connection Restored" toasts.

So we either:
1. Say "too bad" and force them all to be the default colour
2. Don't officially support the attribute in docs but behind the scenes we keep allowing it
3. Adjust the design somehow to allow for this in edge cases

@geurts @cheics-d2l any thoughts?

Reply:
We had a previous [conversation here in Slack](https://d2l.slack.com/archives/G0147DRG8CQ/p1589400472017500), which seemed to imply:

2. Don't officially support the attribute in docs but behind the scenes we keep allowing it

As far as colour - I think we could probably get away with changing all the Blue -> Green, but not so much with changing Red -> Blue, since Red is laden with different meaning.

Maybe an option 1/2 combo? 

- Say "too bad" and force all GREEN toasts to blue, don't officially support the attribute but allow legacy toasts to still be red

Again - the quizzing use case is more along "we need a pattern for when a connection is lost and restored", rather than usage of the specific toast pattern.

@geurts Thoughts?

Reply:
I am okay with approach # 2. I am also okay with #1.5 if it's not a bunch of extra effort. 

(I guess I need to follow my notifcations in gmail more closely, sorry guys - Scy alerted me to this thread or I wouldn't have found it)